judge/forms.py

```diff
@@ -116,11 +117,27 @@ class Meta:
 
 
 class ProblemEditForm(ModelForm):
+    statement_file = forms.FileField(
+        required=False,
+        validators=[FileExtensionValidator(allowed_extensions=settings.PDF_STATEMENT_SAFE_EXTS)],
+        help_text=_('Maximum file size is %s.') % filesizeformat(settings.PDF_STATEMENT_MAX_FILE_SIZE),
```

Comment:
To support the pdf file type in front end, please add
`widget=forms.FileInput(attrs={'accept': 'application/pdf'})`

Reply:
I fixed it myself